_includes/pbc/all/install-features/202407.0/install-the-acl-feature.md

```diff
@@ -237,6 +239,28 @@ Make sure the configured data has been added to the following database tables:
 
 {% endinfo_block %}
 
+5. Synchronize the ACL entities for the merchants and their users:
+
+```bash
+console acl:entity:synchronize
+```
+
+{% info_block warningBox "Verification" %}
+
+Make sure the synchronized data has been added to the following database tables:
+* `spy_acl_entity_rule`
+* `spy_acl_entity_segment`
+* `spy_acl_entity_segment_merchant`
+* `spy_acl_entity_segment_merchant_user`
+* `spy_acl_group`
+* `spy_acl_groups_has_roles`
+* `spy_acl_role`
+* `spy_acl_rule`
+
+With a multi-merchant environment, make sure the ACL entities are synchronized for each merchant.
+If the ACL entities have already been synchronized, the synchronization process doesn't create duplicate entries.
```

Comment:
Is the last sentence part of verification?

Reply:
Yes, it is.